### PR TITLE
GitHub Discussions Announcement

### PR DESCRIPTION
**Pull request by kpodemski**
😎

### CODE REVIEW

File: content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "GitHub Discussions are now enabled in the PrestaShop Project!"
+subtitle: "A better place for the community to discuss the project"
+date:   2023-01-12 14:00:00
+authors: [ PrestaShop ]
```
Comment (eternoendless) [inline content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md]: I believe you use "I" somewhere
```suggestion
authors: [ kpodemski ]
```

File: content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+icon: icon-chat
+tags: [ news, contribute, community, ]
```
Comment (eternoendless) [inline content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md]:
```suggestion
tags: [ news, contribute, community ]
```

File: content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## What happens to the Issues?
+
+From now on, **issues should be used for bug reports only**. Feature requests are, in fact, “ideas” which should be discussed in the appropriate _Discussions_ section.
+
+There are over 1,200 open feature requests currently on GitHub issues. To avoid polluting the newly created Discussions space with hundreds of topics, project members will progressively go through all existing feature requests and either close them or migrate them over to the Discussions platform, based on engagement criteria like the number of reactions and duplicate reports. Rest assured that nothing will be lost in the process: migrated topics will keep a link to the previous issue, and all previous comments can be retrieved even if the original issue is closed. 
+
+If a feature request is closed, it doesn’t mean it cannot be done and never will never be: it just means that at right now, it hasn’t gained enough traction to get done by anyone. If you think one of those is still worth doing, consider submitting it again as a Discussion, and see if it is upvoted by the community. Remember: ideas are most useful when people get interested in making them come true.
```
Comment (eternoendless) [inline content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md]:
```suggestion
If a feature request is closed, it doesn’t mean it cannot be done and never will never be: it just means that right now, it hasn’t gained enough traction to get done by anyone. If you think one of those is still worth doing, consider submitting it again as a Discussion, and see if it is upvoted by the community. Remember: ideas are most useful when people get interested in making them come true.
```

**Review by eternoendless — APPROVED** [review]
OK for me, waiting for @coffrep 's review

File: content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+**Announcements**
+
+This is where we share news, updates, and important information about the PrestaShop project (e.g., new releases). Please check back here regularly to stay informed about what's going on with the project.
+
+**Ideas**
+
+This is a place for open discussion about ideas you find valuable for the PrestaShop project. Feel free to share your thoughts on how PrestaShop could be improved for everyone. 
+
+You can also browse through other ideas and vote on your favorites by giving them a thumb-up. This helps project members and contributors understand which ideas are most popular, and the higher visibility will encourage the community to engage in them - and perhaps even submit a pull request!
```
Comment (coffrep) [inline content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md]:
```suggestion
You can also browse through other ideas and vote on your favorites by giving them a thumbs-up. This helps project members and contributors understand which ideas are most popular, and the higher visibility will encourage the community to engage in them - and perhaps even submit a pull request!
```

File: content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+## What happens to the Issues?
+
+From now on, **issues should be used for bug reports only**. Feature requests are, in fact, “ideas” which should be discussed in the appropriate _Discussions_ section.
+
+There are over 1,200 open feature requests currently on GitHub issues. To avoid polluting the newly created Discussions space with hundreds of topics, project members will progressively go through all existing feature requests and either close them or migrate them over to the Discussions platform, based on engagement criteria like the number of reactions and duplicate reports. Rest assured that nothing will be lost in the process: migrated topics will keep a link to the previous issue, and all previous comments can be retrieved even if the original issue is closed. 
+
+If a feature request is closed, it doesn’t mean it cannot be done and never will never be: it just means that right now, it hasn’t gained enough traction to get done by anyone. If you think one of those is still worth doing, consider submitting it again as a Discussion, and see if it is upvoted by the community. Remember: ideas are most useful when people get interested in making them come true.
```
Comment (coffrep) [inline content/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project.md]:
```suggestion
If a feature request is closed, it doesn’t mean it cannot be done and never will be: it just means that right now, it hasn’t gained enough traction to get done by anyone. If you think one of those is still worth doing, consider submitting it again as a Discussion, and see if it is upvoted by the community. Remember: ideas are most useful when people get interested in making them come true.
```

**Review by coffrep — APPROVED** [review]
Hi - great post! I've added two minor edits, let me know if the second edit makes sense or not.

**Pull request merged**